### PR TITLE
bug-fix for bright objects without saturated pixels

### PR DESCRIPTION
**Pull request by jchiang87**

### CODE REVIEW

File: imsim/diffraction_fft.py
Comment (rmjarvis) [inline imsim/diffraction_fft.py:165]: Probably want to add a unit test that hits this line.

**Review by rmjarvis — CHANGES_REQUESTED** [review]

File: tests/test_diffraction_fft.py
Comment (g-braeunlich) [inline tests/test_diffraction_fft.py:470]:
```suggestion
    np.testing.assert_allclose(image, 0.)
```

**Review by g-braeunlich — CHANGES_REQUESTED** [review]
Thx for the fix / unit test!
Just one small suggestion.

File: tests/test_diffraction_fft.py
Comment (jchiang87) [reply tests/test_diffraction_fft.py:470]: Good idea! Thanks.

Comment (jchiang87) [reply tests/test_diffraction_fft.py:470]: @g-braeunlich I committed your change without using the "Commit suggestion" button, so I guess I still need your approval before merging.  Could you please approve?

**Review by g-braeunlich — APPROVED** [review]
:star:

**Pull request merged**
